### PR TITLE
feat(console): add tenant suspended page

### PR DESCRIPTION
**Pull request by xiaoyijun**
## Summary

Display a suspended page when a tenant is suspended.

### Updates
- Reorg the `AppConent` folder according to our [console convention](https://www.notion.so/silverhand/Console-Convention-20de50cfa97d4d6d92d3f19862adae2e)
- Move the `TenantEnvTag` and `TopBar` component into the `components` folder since it's shared in multiple pages
- Support suspended tag in the tenant selector item

### Todo
- Add UI tests for this feature in the cloud repo.

## Testing

image

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests (need to add in the cloud repo)
- [ ] docs

OR

- [x] This PR is not applicable for the checklist

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/console/src/containers/AppContent/TenantSuspendedPage/index.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+      <div className={styles.title}>
+        <DynamicT forKey="tenants.tenant_suspended_page.title" />
+      </div>
+      <div className={styles.description}>
+        <DynamicT forKey="tenants.tenant_suspended_page.description_1" />
+      </div>
+      <div className={styles.description}>
+        <DynamicT forKey="tenants.tenant_suspended_page.description_2" />
+      </div>
+      <a href={contactEmailLink} target="_blank" className={styles.linkButton} rel="noopener">
```
Comment (simeng-li) [inline packages/console/src/containers/AppContent/TenantSuspendedPage/index.tsx:28]: do we need target="_blank" here ?

File: packages/console/src/containers/AppContent/index.tsx [hunk 2]
```diff
@@ -1,38 +1,46 @@
...
 
-import Topbar from './components/Topbar';
+import TenantSuspendedPage from './TenantSuspendedPage';
 import * as styles from './index.module.scss';
 import { type AppContentOutletContext } from './types';
 
 export default function AppContent() {
   const { isLoading } = useUserPreferences();
+  const { currentTenant } = useContext(TenantsContext);
+  const isTenantSuspended = isCloud && currentTenant?.isSuspended;
```
Comment (simeng-li) [inline packages/console/src/containers/AppContent/index.tsx:24]: nit: technically no need to check `isCloud` here

**Review by simeng-li — COMMENTED** [review]
Just double check. With this suspend page implemented, in a normal flow, AC user won't see the tenant suspend api error message toast anymore.

File: packages/console/src/containers/AppContent/TenantSuspendedPage/index.tsx (hunk 1 above)
Comment (xiaoyijun) [reply packages/console/src/containers/AppContent/TenantSuspendedPage/index.tsx]: Good call, it's redundant, removed.

File: packages/console/src/containers/AppContent/index.tsx (hunk 2 above)
Comment (xiaoyijun) [reply packages/console/src/containers/AppContent/index.tsx:24]: This feature is cloud-only.
Although in the open-source version, the `isSuspended` field in the `currentTenant` data is set to `false`, but it is merely dummy data.
Therefore, we use `isCloud` here to ensure that this functionality does not accidentally activate due to changes in dummy data.

**Comment by xiaoyijun** [comment]
> Just double check. With this suspend page implemented, in a normal flow, AC user won't see the tenant suspend api error message toast anymore.

Yes.

**Pull request merged**